mkdocs.yml

```diff
@@ -0,0 +1,49 @@
+site_name: Perseo Context-aware Complex Event Processing
+site_description: Perseo Documentation
+repo_url: https://github.com/Ficodes/perseo-fe
+theme: readthedocs
+extra_css: ['https://fiware.org/style/fiware_readthedocs.css', "https://fiware.org/style/fiware_readthedocs_core.css"]
```

Comment:
The extra css is incorrect - should be `https://www.fiware.org/style/fiware_readthedocs_processing.css`

Reply:
ok. Fixed in ef4582b